typescript/src/entity.ts

```diff
@@ -1,10 +1,14 @@
 import 'reflect-metadata';
+import { apiFetch } from './request';
 
 
-export abstract class Entity {
```

Comment:
`abstract` has to be removed, to make the `InstanceType` hack work, since otherwise `Entity` isn't newable.